xmtp/src/builder.rs

```diff
@@ -56,8 +66,8 @@ where
         self
     }
 
-    pub fn wallet_address(mut self, wallet_address: String) -> Self {
-        self.wallet_address = Some(wallet_address);
+    pub fn wallet_address(mut self, wallet_address: &str) -> Self {
```

Comment:
Curious why this change was necessary, given it's being converted back to a string